external-api/src/v1/controllers/acbs.controller.ts

```diff
@@ -169,9 +200,11 @@ const amendAcbsFacility = async (amendment: Amendment) => {
   const hasAmendment = amendment.coverEndDate || amendment.amount;
 
   if (hasAmendment) {
+    const url = `${process.env.AZURE_ACBS_FUNCTION_URL}/api/orchestrators/acbs-amend-facility`;
```

Comment:
Can we declare it in a constant at the top of the file? It seems repetitive

Reply:
The URL differs per function, or do you mean declare the domain as constant?

Reply:
Yes, the domain